app/uk/gov/hmrc/agentservicesaccount/views/pages/assistant/your_account.scala.html

```diff
@@ -79,17 +81,15 @@
         @p("your.account.your.groups.off.p")
     }
 
-
     <div id="bottom-panel">
         <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
         @h2("your.account.your.admins.h2")
-        @a("your.account.your.admins.a", href = routes.AgentServicesController.administrators.url)
```

Comment:
I don't see this key removed from the messages files... We should remove it if it's not used anywhere else and being replaced with "administrators.link-text"

Reply:
ah, well spotted! Thx